config/master/manifests.jsonnet

```diff
@@ -297,40 +297,11 @@ local metricsHelper = {
       {
         apiGroups: [""],
         resources: [
-          "nodes",
```

Comment:
This needs to be updated too - https://github.com/aws/amazon-vpc-cni-k8s/blob/master/charts/cni-metrics-helper/templates/clusterrole.yaml